packages/fxa-auth-server/lib/senders/emails/partials/paymentProvider/index.mjml

```diff
@@ -2,14 +2,17 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
-<mj-text css-class="text-body margin-top">
-  <% if (payment_provider === 'paypal') { %>
+<% if (payment_provider === 'paypal') { %>
+  <mj-text css-class="text-body margin-top">
     <span data-l10n-id="payment-method">Payment Method: </span>
     <span>PayPal</span>
-  <% } else { %>
+  </mj-text>
+  <%# Stripe is the default payment provider, but check for truthiness in case of coupons %>
+<% } else if (cardType && lastFour) { %>
```

Comment:
I suppose it's just a preference to have this within an else statement.... 🤔

Reply:
Do you mean using `if else` instead of another `if`? Yeah, I could've gone with just `if` since they should never both be true, this just feels slightly better since it doesn't have to go into the 2nd set of logic if the first `if` is truthy.